geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -2,6 +2,8 @@
 /// the GeyserPlugin trait to work with the runtime.
 /// In addition, the dynamic library must export a "C" function _create_plugin which
 /// creates the implementation of the plugin.
+use solana_sdk::pubkey::Pubkey;
```

Comment:
Follow our coding style, put them in "use {...}";

Reply:
Corrected